docs/content/system-props.mdx

```diff
@@ -6,7 +6,7 @@ import {PropsList, COMMON, LAYOUT, BORDER, TYPOGRAPHY, FLEX, POSITION, GRID} fro
 
 <Note variant="warning">
 
-System props are deprecated in all components except [Box](/Box). Please use the [`sx` prop](/overriding-styles) instead.
+System props are deprecated in all components except [Box](/Box) and [Text](/Text). Please use the [`sx` prop](/overriding-styles) instead.
```

Comment:
TIL. Why are these two components the exceptions?

Reply:
As I understand it, the thought is that these two components are the basic building blocks of styling — whereas other components _should_ be opinionated about their styling and overriding their styles shouldn't be as dominant a part of their API.

I lean toward this approach but I don't think it would be the end of the world if we went with `sx` everywhere.

cc @colebemis who I think is on the fence about it.